OpenRA.Mods.Common/Traits/Render/Hovers.cs

```diff
@@ -46,15 +68,17 @@ public class Hovers : ConditionalTrait<HoversInfo>, IRenderModifier, ITick
 		readonly int stepPercentage;
 		readonly int fallTickHeight;
 
-		int ticks = 0;
-		WVec worldVisualOffset = WVec.Zero;
+		int ticks;
+		WVec worldVisualOffset;
 
 		public Hovers(HoversInfo info, Actor self)
 			: base(info)
 		{
 			this.info = info;
-			this.stepPercentage = 256 / info.Ticks;
-			this.fallTickHeight = (info.InitialHeight + info.OffsetModifier) / info.FallTicks;
+			stepPercentage = 256 / info.Ticks;
+
+			// fallTickHeight must be at least 2 to avoid a DivideByZeroException and other potential problems when trait is disabled.
```

Comment:
Can you explain this? I don't understand how we will get a divide by zero if this value is 1.
`var fallTicks = worldVisualOffset.Z / fallTickHeight - 1;` should divide first, then subtract one.

Reply:
Right, but the value still needs to be at least 1 to avaoid said DBZ Exception. Updated accordingly.